### ISSUE TITLE
extend Kotlin default imports

### ISSUE THREAD

**Issue by linux-china** [cbeust/kash#8]
The Followings are  default Imports of Kotlin, could the use add more global imports for kash. 
```
kotlin.*
kotlin.annotation.*
kotlin.collections.*
kotlin.comparisons.* (since 1.1)
kotlin.io.*
kotlin.ranges.*
kotlin.sequences.*
kotlin.text.*
```

For example if you use kclock in kash: 

```
// good 
val now = DateTime.now()
//bad
val com.soywiz.korlibs.klock:klock-jvm.DateTime.now()
```

too much code for package name, and nobody can remember the package name.  

With global imports introduced, and I can add "com.soywiz.korlibs.klock:klock-jvm" in global imports, and use DateTime directly.

**Comment by cbeust** [comment]
Yes, take a look at the [TODO](https://github.com/cbeust/kash/blob/master/TODO.md) and you'll see I have import management listed there.
